app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/apiClientSidebarCollapse/apiClientSidebarCollapse.scss

```
@@ -0,0 +1,114 @@
+  .ant-collapse-item {
+    width: 100%;
+
+    .ant-collapse-header {
+      height: 28px;
+      padding: 2px 4px;
+      border-radius: 4px !important;
+      user-select: none;
+      display: flex;
+      align-items: center;
+      gap: var(--requestly-space-2, 4px);
+      color: var(--requestly-color-text-subtle);
```

Stylelint (17.3.0)

[error] Unexpected duplicate "color" (declaration-block-no-duplicate-properties)

(declaration-block-no-duplicate-properties)